flutter_local_notifications/lib/src/flutter_local_notifications_plugin.dart

```diff
@@ -247,4 +248,13 @@ class FlutterLocalNotificationsPlugin {
     return FlutterLocalNotificationsPlatform.instance
         ?.pendingNotificationRequests();
   }
+
+  Future<bool> createNotificationChannel(AndroidNotificationChannel notificationChannel) async {
```

Comment:
See comment made previously on making this return Future<void> as the Android code never returns false. The return null statement will be redundant after making those changes